raiden_installer/web.py

```diff
@@ -703,6 +710,21 @@ def post(self, configuration_file_name):
         )
 
 
+class GasPriceHandler(APIHandler):
+    def get(self, configuration_file_name):
+        configuration_file = RaidenConfigurationFile.get_by_filename(configuration_file_name)
+        account = configuration_file.account
+        try_unlock(account)
+        web3 = make_web3_provider(configuration_file.ethereum_client_rpc_endpoint, account)
```

Comment:
This looks inefficient. Is there no way to creae the web3 provider only once and use it for all requests? This way for every request you reload all blocks required for the gas estimation instead of caching them.

Reply:
We only call the endpoint once. Tornado is supposed to be stateless, therefore we must create our objects on every call. There are workarounds for it, but since we call this endpoint only once should be sufficient for now

Reply:
Seems like you can use `initialize` (https://www.tornadoweb.org/en/stable/guide/structure.html#the-application-object), but when it's only called once that's no problem anyways.